Drivers/rc_receiver/Src/sbus_receiver.cpp

```diff
@@ -97,4 +98,12 @@ void SBUSReceiver::cast_rccontrol()
     if(channel_value > SBUS_RANGE_MAX)
         channel_value = SBUS_RANGE_MAX;
     return static_cast<float>((channel_value - SBUS_RANGE_MIN) * (100.0f / SBUS_RANGE_RANGE));
- }
\ No newline at end of file
+ }
+
+ bool SBUSReceiver::getIsDataNew(){
+    return is_data_new_;
+ }
+
+ void SBUSReceiver::setIsDataNew(bool newVal){
+    this->is_data_new_ = newVal;
```

Comment:
What is the purpose of checking for new data? Just missing context && haven't been following

Reply:
Failsafe, in case the receiver gets unplugged